usagov/vote-gov-drupal
# Allow svg and webp images from theme directory.

## Jira ticket

N/A

## Description

Include SVG and WEBP images in robots.txt for production.

## Deployment and testing

### Post-deploy steps

1. Change your settings.local.php to include `$config['config_split.config_split.production']['status'] = TRUE;`
2. Run `lando retune`

### QA/Testing instructions

1. Login as an administrator
2. Goto /admin/config/search/robotstxt and confirm entries for svg and webp images.

## Checklist for the Developer




## Checklist for the Peer Reviewers
